DEPLOY.md

```diff
@@ -47,6 +47,8 @@ server {
                 proxy_redirect     off;
                 proxy_set_header Host $host;
                 proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+		proxy_connect_timeout 300s;
```

Comment:
hard tabs have revealed that the rest of this was annoyingly misindented. if you feel like fixing that, sure, if not, I'll hit the button.